lib/screens/app/app.dart

```diff
@@ -113,9 +114,9 @@ class _AppState extends State<App> with WidgetsBindingObserver {
             if (pageCommand is BottomBarNavigateToIndex) {
               _pageController.jumpToPage(pageCommand.index);
             } else if (pageCommand is ShowErrorMessage) {
-              SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
+              eventBus.fire(ShowSnackBar(pageCommand.message));
             } else if (pageCommand is ShowMessage) {
-              SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
+              eventBus.fire(ShowSnackBar(pageCommand.message));
```

Comment:
@gguijarro-c-chwy 
This is a pattern that I have seen about 3 or 4 times there are the 2 types of show message 
(`ShowErrorMessage`, `ShowMessage`) and both launch the **same type** of snack bar.

We are using the same gray snack bar for error and info I don't know if this should continue like this or we should ask Jason for a specific design/color to differentiate the error ones from the info ones.

Reply:
we should ask Jason for designs for each. Or Luigi